torchchat/usages/browser.py

```diff
@@ -53,15 +137,20 @@ def get_streamed_completion(completion_generator):
                 state="complete",
             )
 
-        response = st.write_stream(
-            get_streamed_completion(
-                client.chat.completions.create(
-                    model="llama3",
-                    messages=st.session_state.messages,
-                    max_tokens=response_max_tokens,
-                    stream=True,
+        try:
+            response = st.write_stream(
+                get_streamed_completion(
+                    client.chat.completions.create(
+                        model="llama3",
```

Comment:
remind me why this is set to llama3?

Reply:
For now, this could be set to anything - I just set the string so the API doesn't complain. 

Since loading a model is very expensive, we just load whatever the server is launched with. 

I could add a task for selecting a model from a list since the models endpoint is already implemented.

Reply:
Not in this PR, but we should comment this